Intersect.Server/Database/Item.cs

```diff
@@ -20,44 +21,24 @@ public Item()
         {
         }
 
-        public Item(Guid itemId, int quantity, bool incStatBuffs = true) : this(
-            itemId, quantity, null, null, incStatBuffs
+        public Item(Guid itemId, int quantity, ItemProperties ItemProperties = null) : this(
+            itemId, quantity, null, null, ItemProperties
         )
         {
         }
 
-        public Item(Guid itemId, int quantity, Guid? bagId, Bag bag, bool includeStatBuffs = true)
+        public Item(Guid itemId, int quantity, Guid? bagId, Bag bag, ItemProperties itemProperties = null)
         {
             ItemId = itemId;
             Quantity = quantity;
             BagId = bagId;
             Bag = bag;
-
-            var descriptor = ItemBase.Get(ItemId);
-            if (descriptor == null || !includeStatBuffs)
-            {
-                return;
-            }
-
-            if (descriptor.ItemType != ItemTypes.Equipment)
-            {
-                return;
-            }
-
-            for (var i = 0; i < (int) Stats.StatCount; i++)
-            {
-                // TODO: What the fuck?
-                StatBuffs[i] = Randomization.Next(-descriptor.StatGrowth, descriptor.StatGrowth + 1);
-            }
+            ItemProperties = itemProperties != null ? itemProperties : new ItemProperties(ItemId);
         }
 
         public Item(Item item) : this(item.ItemId, item.Quantity, item.BagId, item.Bag)
         {
-            for (var i = 0; i < (int) Stats.StatCount; i++)
-            {
-                StatBuffs[i] = item.StatBuffs[i];
-            }
-
+            ItemProperties = item.ItemProperties;
```

Comment:
This should be cloning the item properties as well.

Something like `ItemProperties = new ItemProperties(item.ItemProperties);` or `ItemProperties = item.ItemProperties.Clone()`.